Integrations/GoogleDocs/GoogleDocs.py

```diff
@@ -0,0 +1,395 @@
+        demisto.results({
+            'Type': entryTypes['error'],
+            'ContentsFormat': formats['text'],
+            'Contents': 'Enter required_revision_id or target_revision_id but not both'
+        })
+        return
+    elif required_revision_id:
+        write_control['writeControl']["requiredRevisionId"] = required_revision_id
+    elif target_revision_id:
+        write_control['writeControl']["targetRevisionId"] = target_revision_id
+
+    payload = {**payload, **write_control}
+
+    for action_type, params in actions.items():
+        request = globals()[ACTION_TO_FUNCTION[action_type]](action_type, *params)
+        payload["requests"].append(request)
+
+    service.documents().batchUpdate(documentId=document_id, body=payload).execute()
+    document = get_document(service, document_id)
+    return document
+
+
+def create_document_command(service):
+    args = demisto.args()
+    title = args.get('title')
+
+    payload = {
+        "title": title,
+    }
+
+    document = service.documents().create(body=payload).execute()
+    return document
+
+
+def get_document_command(service):
+    args = demisto.args()
+    document_id = args.get('document_id')
+    document = get_document(service, document_id)
+    return document
+
+
+def get_document(service, document_id):
+    document = service.documents().get(documentId=document_id).execute()
+    return document
+
+
+def main():
+    LOG('Command being called is %s' % (demisto.command()))
+    proxy = demisto.params().get('proxy')
+    disable_ssl = demisto.params().get('insecure', False)
+    service_account_credentials = json.loads(demisto.params().get('service_account_credentials'))
+    if demisto.command() == 'test-module':
+        try:
+            get_client(service_account_credentials, SCOPES, proxy, disable_ssl)
+            demisto.results('ok')
+        except Exception as e:
+            return_error("Failed to execute test. Error: {}".format(str(e)), e)
+
+    try:
+        service = get_client(service_account_credentials, SCOPES, proxy, disable_ssl)
+        if demisto.command() == 'update_document':
+            res = batch_update_document_command(service)
+        elif demisto.command() == 'create_document':
+            res = create_document_command(service)
+        elif demisto.command() == 'get_document':
+            res = get_document_command(service)
+        else:
+            return_error("Command {} does not exist".format(demisto.command()))
+            return
+
+        ec = {
+            'GoogleDocs(val.DocumentId && val.DocumentId == DocumentId.Query)': {
+                'RevisionId': res['revisionId'],
+                'DocumentId': res['documentId'],
+                'Title': res['title']
+            }
+        }
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['markdown'],
+            'Contents': ec,
+            'HumanReadable': json.dumps(ec),
```

Comment:
We'll probably need to make a better human readable, let's revisit this point after we'll sit on the integration together